src/Components/Facility/HospitalList.tsx

```diff
@@ -35,6 +35,16 @@ export const HospitalList = () => {
   } = useFilters({
     limit: 14,
   });
+
+  useEffect(() => {
+    if (!qParams.state) {
+      advancedFilter.removeFilters(["district", "local_body"]);
```

Comment:
Would removing the `district` filter here cause this useEffect hook to re-run and then again attempt to remove local_body filter (at the 2nd if statement)? Seems to work fine though, just something to note 🤔  

cc: @rithviknishad

Reply:
The `removeFilters` constructs the updated object after removing all the keys specified and does the `setQueryParams` with the new object in one go. So this wouldn't be an issue here as multiple keys are removed in a single navigation.

Ideally, we should have a way to specify dependent filter keys to the args of `useFilters` itself. But we could skip it for now since we don't have so many dependent filters right now.